### ISSUE TITLE
FR: make this prompt optional/off from config

### ISSUE THREAD

**Issue by robmarkcole** [ESA-PhiLab/iris#44]
As option in config, to not see this:

<img width="664" alt="image" src="https://github.com/ESA-PhiLab/iris/assets/11855322/706f0403-e0c3-4e9c-a420-ebe1c414634d">

**Comment by aliFrancis** [comment]
Yeah, need to implement this. Might be a bit more complex than just setting it to not appear, as I'll need to decide whether to write default values in lieu of any response, or to change the way the database tables are structured. The latter probably makes more sense but will take more time.

**Comment by robmarkcole** [comment]
I'd mark this as low priority
